Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -25,8 +25,16 @@ function vtkMouseRangeManipulator(publicAPI, model) {
   function processDelta(listener, delta) {
     const oldValue = listener.getValue();
 
-    // Apply scale and cached delta to current delta
-    const newDelta = delta * listener.scale + incrementalDelta.get(listener);
+    // if exponential scroll is enabled, we raise our scale to the
+    //  exponent of the net delta of the interaction. The further away
+    // the user's cursor is from the start of the interaction, the more
+    // their movements will effect the value.
+    const scalingFactor = listener.exponentialScroll
+      ? listener.scale ** Math.log(Math.abs(model.interactionNetDelta) + 2)
```

Comment:
When the interactionNetDelta is 0, we get log(2) using the natural log. What is your intended scaling factor when interactionNetDelta is 0? scale^1? Or 1?

Reply:
Ah yea good point, intended value would be n^1

Reply:
I'll update

Reply:
I'll update

Reply:
I think you'll want to do `Math.log2()` instead of `model.interactionNetDelta !== 0` to achieve the proper effect. That way  `Math.log2(0 + 2)` resolves to `1` and consequently scalingFactor will resolve to `listener.scale ^ 1`.